landing-page/src/components/landing-page-network.ts

```diff
@@ -111,17 +113,20 @@ class LandingPageNetwork extends LitElement {
 
     this._getNetworkInfoError = false;
 
+    const primaryInterface = data.interfaces.find(
+      (intf) => intf.primary && intf.enabled
+    );
+    if (primaryInterface) {
+      this._dnsPrimaryInterfaceNameservers = [
+        ...(primaryInterface.ipv4?.nameservers || []),
+        ...(primaryInterface.ipv6?.nameservers || []),
+      ].join(", ");
+
+      this._dnsPrimaryInterface = primaryInterface.interface;
```

Comment:
`this._dnsPrimaryInterface` should be cleared if we fetch supervisor network info and there is no primary interface